src/array.cr

```diff
@@ -1549,6 +1549,17 @@ class Array(T)
     dup.sort!
   end
 
+  # Returns an array with all elements in the collection sorted based on the return value of the block given.
+  #
+  #```
+  # a = [3, 1, 2]
+  # b = a.sort { |a,b| b <=> a }
```

Comment:
Space after the comma

You should run the crystal formatter tool to fix those spacing for you!